events/reporter.go

```diff
@@ -91,6 +91,31 @@ func ReportNewActivation(activation *types.ActivationTx) {
 	}
 }
 
+// SubscribeToRewards subscribes a channel to rewards events
+func SubscribeToRewards(bufsize int) chan Reward {
+	mu.RLock()
+	defer mu.RUnlock()
+	if reporter != nil {
+		newChan := make(chan Reward, bufsize)
+		reporter.rewardsSubs[newChan] = struct{}{}
+		return newChan
+	}
+	return nil
+}
+
+// UnsubscribeFromRewards unsubscribes a channel from rewards events
+// do we need to close the subscriber channel here?
+func UnsubscribeFromRewards(subscriber chan Reward) {
```

Comment:
the `subscriber` arg should be renamed `subscription` or `subscriptionChannel` or something like that